t/core/utils.t

```diff
@@ -68,3 +68,19 @@ qr/random seed \d+\ntwice: false/
 GET /t
 --- no_error_log
 [error]
+
```

Comment:
The test case did not cover empty configuration of resolver

Reply:
added.

Reply:
No, I means the `config.yaml` file

Reply:
Here we do not need to set the `dns_resolver` in `config.yaml`. Through the `set_resolver` function in the ʻutils` file. We can set `resolvers` to be empty for testing.

Reply:
Here we do not need to set the `dns_resolver`  in `config.yaml`. Through the `set_resolver` function in the `utils`  file. We can set `resolvers` to be empty for testing.